spec/models/message_draft_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe MessageDraft do
+  context 'with valid attributes' do
+    subject { described_class.new(params) }
+
+    let(:params) { attributes_for :message }
+
+    it 'aliases draft_id for id' do
+      expect(subject.draft_id).to eq(subject.id)
```

Comment:
I don't think this is necessary. The object is a MessageDraft, id is descriptive enough as it is, don't need this alias.